doc/src/sphinx/Futures.rst

```diff
@@ -168,7 +168,7 @@ The following retries a request infinitely should it fail with a
 
 	def fetchUrl(url: String): Future[HttpResponse]
```

Comment:
actually, since we no longer expose netty types in the public finagle API, how about we change this to http.Response?